src/drf_yasg/generators.py

```diff
@@ -409,9 +409,9 @@ def get_path_parameters(self, path, view_cls):
             if getattr(view_cls, 'lookup_field', None) == variable and attrs['type'] == openapi.TYPE_STRING:
                 attrs['pattern'] = getattr(view_cls, 'lookup_value_regex', attrs.get('pattern', None))
 
-            if model_field and model_field.help_text:
+            if model_field and hasattr(model_field, 'help_text') and model_field.help_text:
```

Comment:
``hasattr(model_field, 'help_text') and model_field.help_text`` could be replaced by ``getattr(model_field, 'help_text', '')``